src/profile/helpers/editData.ts

```diff
@@ -150,17 +182,254 @@ function pickSources(
   }
 }
 
+export function cloneAndMutateItem(
```

Comment:
Did not find any usages outside of this file, so maybe the export is unnecessary?

Reply:
Removed.